addons/sale_coupon/data/ir_cron_data.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" ?>
+<odoo noupdate="1">
+    <record id="expire_coupon_cron" model="ir.cron">
```

Comment:
I'd have just added it in `addons/sale_coupon/data/sale_coupon_email_data.xml` as a real backport.
As it is a backport, better to have it at the same place between versions IMO https://github.com/odoo/odoo/pull/34620/commits/7092d0ae9b02a2b12d953e3d0bc3cc77fa6df5e1#diff-9c1f2adf756382bb0177ddf1d5a0e2d82e40deee79453877251e17c1fc545c39R105

Reply:
Done!